CODEOWNERS

```
@@ -0,0 +1,4 @@
+# release configuration
+
+/.release/                              @hashicorp/release-engineering @hashicorp/github-secure-vault-ecosystem
+/.github/workflows/build.yml            @hashicorp/release-engineering @hashicorp/github-secure-vault-ecosystem
```

```suggestion
/.release/                              @hashicorp/release-engineering @hashicorp/vault-ecosystem
/.github/workflows/build.yml            @hashicorp/release-engineering @hashicorp/vault-ecosystem
```

Did you mean `vault-ecosystem` instead? The team in the PR currently does not have any of the vault eco team members in it, just a bot account.

---

The bot is used for CRT pipeline builds - without this, any changes to codeowners impacted files would fail in the pipeline, even if properly approved.